packages/uploadthing/src/internal/types.ts

```diff
@@ -26,7 +26,7 @@ type ResolverOptions<TParams extends AnyParams> = {
   file: UploadedFile;
 };
 
-export type AnyRuntime = "app" | "pages" | "web";
+export type AnyRuntime = "app" | "pages" | "web" | "nuxt";
```

Comment:
seems this flag is never used to do anything?

Reply:
If I properly understand the internal type system, it will be required to properly type middleware arguments